src/autopas/utils/logging/Logger.h

```diff
@@ -6,6 +6,7 @@
 
 #pragma once
 
+#include <spdlog/fmt/bundled/ranges.h>
```

Comment:
after including this, lots of calls to `arrayutils::to_string` could, theoretically, be removed.
Would result in different formatting, though, and might require the adaption of some scripts.

Reply:
and is out of scope for this PR